zenodo/modules/records/serializers/bibtex.py

```diff
@@ -301,6 +301,7 @@ def _fetch_fields(self, req_fields, opt_fields=None, ign_fields=None):
 
     def _format_output_row(self, field, value):
         out = ""
+        field = field.rstrip()
```

Comment:
Why not plain `.strip()`?

Reply:
The goal for this is to eliminate trailing whitespace. This should be enough to fix our error case.
Plain stripping means we are reformatting the screen and creates a complexity since you shouldn't stip all spaces, unless I'm missing something